### PR TITLE
adding typing to  behaviours

### PR DESCRIPTION
**Pull request by AbhinavShah421**

### CODE REVIEW

File: packages/engine/src/scene/behaviors/createBoxCollider.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
+import { Quaternion, Vector3 } from '@etherealjs/core';
```
Comment (speigg) [inline packages/engine/src/scene/behaviors/createBoxCollider.ts:1]: Let’s not pull these types from ethereal :)

Just grab them directly from THREE

File: packages/engine/src/scene/behaviors/createGame.ts [hunk 2]
```diff
@@ -6,7 +6,19 @@ import { GameObject } from "../../game/components/GameObject";
 import { TransformComponent } from '../../transform/components/TransformComponent';
 import { GameMode } from "../../game/types/GameMode";
 
-export const createGame: Behavior = (entity, args: any) => {
+interface gameData {
```
Comment (speigg) [inline packages/engine/src/scene/behaviors/createGame.ts:9]: Capitalize GameData

File: packages/engine/src/scene/behaviors/createMedia.ts [hunk 3]
```diff
@@ -150,3 +150,11 @@ function addInteraction(entity): void {
     });
   }
 }
+
+
+interface volumetricProps{
```
Comment (speigg) [inline packages/engine/src/scene/behaviors/createMedia.ts:155]: In general, please capitalize these interfaces

File: packages/engine/src/scene/behaviors/createMeshCollider.ts [hunk 4]
```diff
@@ -6,18 +6,20 @@ import { ColliderComponent } from '../../physics/components/ColliderComponent';
 import { addColliderWithoutEntity } from '../../physics/behaviors/colliderCreateFunctions';
 import { createNetworkRigidBody } from '../../interaction/prefabs/NetworkRigidBody';
 import { addCollidersToNetworkVehicle } from '../../vehicle/prefabs/NetworkVehicle';
+import { Quaternion, Vector3 } from "@etherealjs/core";
```
Comment (speigg) [inline packages/engine/src/scene/behaviors/createMeshCollider.ts:9]: Again, let’s pull these math types from THREE

**Review by speigg — CHANGES_REQUESTED** [review]

**Review by AbhinavShah421 — COMMENTED** [review]
I have checked in suggested changes please have a look.

**Pull request closed without merging**